osf/models/preprint_provider.py

```diff
@@ -28,6 +28,7 @@ class PreprintProvider(ObjectIDMixin, BaseModel):
     social_twitter = models.CharField(null=True, blank=True, max_length=200)  # max length on prod: 8
```

Comment:
Is there any strategy for keeping these in sync with the html fields? Since they're versioned, if someone changes the html version this won't change and then will be incorrect. Or if someone changes this version via the old API then the html version will be incorrect.